docs/aws_storage.md

```diff
@@ -0,0 +1,41 @@
+# AWS Storage setup instruction for Loki
+
+## Example config via yaml file
+```yaml
+schema_config:
+  configs:
+  - from: 0
+    store: aws
+    object_store: aws
```

Comment:
Could you use `store: dynamodb` and `object_store: s3` to its more explicit please?

Reply:
Yes,  I will be more explicit with this info.  I will base the config from https://github.com/cortexproject/cortex/blob/master/pkg/chunk/storage/factory.go#L127.